src/sentry/api/urls.py

```diff
@@ -1529,6 +1534,16 @@
                     OrganizationMetricsTagDetailsEndpoint.as_view(),
                     name="sentry-api-0-organization-metrics-tag-details",
                 ),
+                url(
+                    r"^(?P<organization_slug>[^/]+)/stacktraces/$",
```

Comment:
Should this have profiling in the path somewhere? Stacktraces are also part of error events.

Reply:
I can prefix all routes I create with profiling to make it clear who it belongs to. Is there a convention for something like this right now?

Reply:
Nothing formal. Other routes follow the pattern of `/organizations/<org-slug>/<thing>/<sub-resource>/` so you could do `/organizations/<org-slug>/profiling/stacktraces/` and fit right in :smile:

Reply:
Added a prefix for every class and endpoint!